docs/conf.in.py

```diff
@@ -270,8 +271,22 @@ def skip(app, what, name, obj, would_skip, options):
     return would_skip
 
 
+def escape_vertical_bars(app, what, name, obj, options, lines: List[str]):
+    # Note - Latex docs also contains Vertical Bar, so we
+    # apply this filtering only to specific classes.
+    if name  in [
+        "open3d.data.RedwoodIndoorLivingRoom1",
+        "open3d.data.RedwoodIndoorLivingRoom2",
+        "open3d.data.RedwoodIndoorOffice1",
+        "open3d.data.RedwoodIndoorOffice2",
+    ]:
+        for index in range(len(lines)):
+            lines[index] = lines[index].replace("|", "\|")
+
+
 def setup(app):
     app.connect("autodoc-skip-member", skip)
+    app.connect("autodoc-process-docstring", escape_vertical_bars)
```

Comment:
The issue comes from vertical bars in python docs which are from http://open3d.org/docs/release/python_api/open3d.data.RedwoodIndoorLivingRoom1.html

I understand its not an elegant solution. Please let me know if it can be improved.